compose-with-lightspeed.yaml

```
@@ -0,0 +1,42 @@
+services:
+  ollama:
+    image: docker.io/ollama/ollama:0.10.0-rc0
+    container_name: ollama
+    volumes:
+      - ollama_data:/root/.ollama
```

```suggestion
      - ${LIGHTSPEED_MODELS_PATH:-ollama_data}:/root/.ollama
```

If this is exposed as an env var, maybe this would avoid the need to modify the version-controlled file and simplify https://github.com/karthikjeeyar/rhdh-local/blob/lightspeed-1.7/additional-config-guides/developer-lightspeed-guide.md#using-your-own-ollama-models-from-your-system ?